package.json

```diff
@@ -27,6 +29,24 @@
     "sass-loader": "^4.0.0",
     "style-loader": "^0.13.1",
     "webpack": "^2.1.0-beta.4",
-    "webpack-md5-hash": "^0.0.5"
+    "webpack-md5-hash": "^0.0.5",
+    "toastr": "2.1.2"
+  },
+  "devDependencies": {
```

Comment:
I'm OK with all this but toastr: is it that important? I don't think it's worth adding it.